spec/abstract-jobs.html

```diff
@@ -1,6 +1,63 @@
 <emu-clause id="sec-abstract-jobs">
   <h1> Abstract Jobs </h1>
 
+  <emu-clause id="sec-weakref-processing-model">
+    <h1>Processing model of WeakRefs and FinalizationGroups</h1>
+
+    <p>
+      This specification does not make any guarantees about garbage collection.
+      Objects which are unreachable from ECMAScript may be released after long periods
+      of time, or never at all. For this reason, this specification uses the term
+      "may" when describing behavior triggered by garbage collection.
+    </p>
+
+    <p>
+      At any given time, if all references to a ECMAScript object are from a
+      [[Target]] field or internal slot, inlcuding the agent's [[KeptAlive]] field,
```

Comment:
Why "[[Target]] field or internal slot" ? If the [[KeptAlive]] slot is treated as the [[Target]], doesn't that mean it fails to keep the object alive until end of job (aka turn)?

Reply:
Note, this text is not new to this PR; I just moved it around (and rephrased the reference to KeptAlive, but not the part you're referring to). I'll think about how to clarify it further, but I think we could do this in a separate patch.